### PR TITLE
RHIDP-4585-Release-Notes-1.4-Final-Fixes

### PR DESCRIPTION
**Pull request by jmagak**
**IMPORTANT: Do Not Merge - To be merged by Docs Team Only**

**Version(s):** 1.4

Add the relevant labels to the Pull Request.
**Issue:** https://issues.redhat.com/browse/RHIDP-4579

### CODE REVIEW

File: modules/release-notes/ref-release-notes-technology-preview.adoc [hunk 1]
```diff
@@ -78,29 +78,32 @@ The following tables list products and their versions compatible with {product-s
...
 
 | `@backstage/config`
 | 1.2.0
 
 | `@backstage/config-loader`
 | 1.9.1
 |===
+
 .Additional resources
 * link:https://issues.redhat.com/browse/RHIDP-5070[RHIDP-5070]
```
Comment (Gerry-Forde) [inline modules/release-notes/ref-release-notes-technology-preview.adoc:109]: @jmagak There are new dynamic plugins included in RHDH 1.4, in previous RNs we listed these (see [here](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html-single/release_notes/index#technology-preview-rhidp-3713))

As far as I know, we are adding the following plugins have been added in this release:

- Signals (frontend) - RH Tech Preview
- Notifications (backend) - RH Tech Preview
- Notifications (frontend) - RH Tech Preview
- Notifications Module Email - RH Tech Preview

@reestr @nickboldt @kadel Can you confirm if these should be mentioned in the RNs?

File: modules/release-notes/ref-release-notes-new-features.adoc [hunk 2]
```diff
@@ -42,26 +42,27 @@ Identify the flavor of Backstage that is running. Default value: `rhdh`
...
 == Authenticating with {rhbk-brand-name}
 
 With this update, you can use {rhbk-brand-name} as an authentication provider. The Keycloak plugin will now support ingesting users and groups with {rhbk-brand-name}. For more details, see link:https://docs.redhat.com/en/documentation/red_hat_build_of_keycloak/26.0/html-single/getting_started_guide/index#getting-started-zip-start-red-hat-build-of-keycloak[Authentication with {rhbk-brand-name}].
 
-
 [id="feature-rhidp-4806"]
-== Ability to install third-party plugins in {product-very-short}
+== [Docs] Document how to install 3rd party plugins to RHDH
+
+= Ability to install third-party plugins in {product-very-short}
```
Comment (Gerry-Forde) [inline modules/release-notes/ref-release-notes-new-features.adoc:60]: @jmagak - this looks badly formatted?
```suggestion
//== [Docs] Document how to install 3rd party plugins to RHDH

== Ability to install third-party plugins in {product-very-short}
```

File: modules/release-notes/ref-release-notes-new-features.adoc [hunk 3]
```diff
@@ -42,26 +42,27 @@ Identify the flavor of Backstage that is running. Default value: `rhdh`
 
 
 [id="feature-rhidp-4419"]
-== Ability to manage PVCs in RHDH Operator
+== [Doc] Document how to manage PVCs in RHDH Operator
+
+= Ability to manage PVCs in RHDH Operator
```
Comment (Gerry-Forde) [inline modules/release-notes/ref-release-notes-new-features.adoc]: @jmagak - this looks badly formatted?
```suggestion
//== [Doc] Document how to manage PVCs in RHDH Operator

== Ability to manage PVCs in RHDH Operator
```

**Review by themr0c — APPROVED** [review]

File: modules/release-notes/ref-release-notes-new-features.adoc [hunk 4]
```diff
@@ -189,7 +186,4 @@ With this update, the `backstage-plugin-catalog-backend-module-logs` is enabled
 [id="feature-rhidp-5218"]
 == {rhsso-brand-name} `7.6` is deprecated as an authentication provider
 
-{rhsso-brand-name} ({rhsso}) `7.6` is deprecated as an authentication provider. You can continue to use {rhsso} until the end of maintenance support. For details, see link:https://access.redhat.com/support/policy/updates/jboss_notes/#p_sso[RHSSO lifecycle dates]. As an alternative, migrate to {rhbk-brand-name} `v24`.
-
-
-
+{rhsso-brand-name} ({rhsso}) `7.6` is deprecated as an authentication provider. You can continue to use {rhsso} until the end of maintenance support. For details, see link:https://access.redhat.com/support/policy/updates/jboss_notes/#p_sso[RHSSO lifecycle dates]. As an alternative, migrate to {rhbk-brand-name} `v24`.
```
Comment (Gerry-Forde) [inline modules/release-notes/ref-release-notes-new-features.adoc:189]: @jmagak Should this not be included in the _Deprecated functionalities_ section?

Comment (jmagak) [reply modules/release-notes/ref-release-notes-new-features.adoc:189]: > @jmagak Should this not be included in the _Deprecated functionalities_ section?

Sure @Gerry-Forde. I will update that.

Comment (Gerry-Forde) [inline modules/release-notes/ref-release-notes-new-features.adoc:189]: Duplicated? It's now in _Deprecated functionality_
```suggestion

```

**Review by Gerry-Forde — APPROVED** [review]

**Pull request merged**